breathe/apidoc.py

```diff
@@ -124,6 +124,20 @@ def recurse_tree(args):
                             compound.get('refid'), args)
 
 
+class TypeAction(argparse.Action):
+    def __init__(self, option_strings, dest, **kwargs):
+        super(TypeAction, self).__init__(option_strings, dest, **kwargs)
+        self.default = TYPEDICT.keys()
+        self.metavar = ','.join(TYPEDICT.keys())
```

Comment:
Is this a required part of the Action API? I can't see it referenced else where in this code.